### PR TITLE
Added a test to verify that listSnapshots response returns zone id

### PR DESCRIPTION
**Pull request by manasaveloori**
Test Snapshot Root Disk ... === TestName: test_01_snapshot_root_disk | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 1 test in 339.130s

OK

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@manasaveloori can you give us the command to which this output fits?

@gauravaradhye can you review this, please? (looks like a lot of code to me)

**Comment by manasaveloori** [comment]
@DaanHoogland 
This is just an enhancement to existing test in the script.
The output is from nose test result of the script.

File: test/integration/smoke/test_snapshots.py [hunk 1]
```diff
@@ -123,65 +126,78 @@ def test_01_snapshot_root_disk(self):
...
         self.assertEqual(
-                            snapshots[0].id,
-                            snapshot.id,
-                            "Check resource id in list resources call"
-                        )
+            snapshots[0].id,
+            snapshot.id,
+            "Check resource id in list resources call"
+        )
+        self.assertNotEqual(
```
Comment (gauravaradhye) [inline test/integration/smoke/test_snapshots.py]: You can directly use assertIsNotNone here.

**Comment by gauravaradhye** [comment]
LGTM except one small change suggested.
Please don't mix intended changes and pep8 fixes, you can add another review request for pep8 fixes subsequently. That will remove the confusion while reviewing.

**Pull request closed without merging**